thewalrus/_hafnian.py

```diff
@@ -251,3 +280,42 @@ def hafnian_repeated(A, rpt, mu=None, loop=False, rtol=1e-05, atol=1e-08):
         return haf_rpt_complex(A, nud, mu=mu, loop=loop)
 
     return haf_rpt_real(A, nud, mu=mu, loop=loop)
+
+
+def banded_loophaf(A):
```

Comment:
We should match the standard, `loop=True` syntax of the other (loop) hafnian functions.
We should also add an input checker.